collections/sentinel-2-l1c.yaml

```diff
@@ -175,51 +239,116 @@ Summaries:
       common_name: coastal
       center_wavelength: 0.4427
       full_width_half_max: 0.021
+      openeo:gsd:
+        value:
+          - 60
+          - 60
+        unit: m
     - name: B02
       common_name: blue
       center_wavelength: 0.4924
       full_width_half_max: 0.066
+      openeo:gsd:
+        value:
+          - 10
+          - 10
+        unit: m
     - name: B03
       common_name: green
       center_wavelength: 0.5598
       full_width_half_max: 0.036
+      openeo:gsd:
+        value:
+          - 10
+          - 10
+        unit: m
     - name: B04
       common_name: red
       center_wavelength: 0.6646
       full_width_half_max: 0.031
+      openeo:gsd:
+        value:
+          - 10
+          - 10
+        unit: m
     - name: B05
       center_wavelength: 0.7041
       full_width_half_max: 0.015
+      openeo:gsd:
+        value:
+          - 20
+          - 20
+        unit: m
     - name: B06
       center_wavelength: 0.7405
       full_width_half_max: 0.015
+      openeo:gsd:
+        value:
+          - 20
+          - 20
+        unit: m
     - name: B07
       center_wavelength: 0.7828
       full_width_half_max: 0.02
+      openeo:gsd:
+        value:
+          - 20
+          - 20
+        unit: m
     - name: B08
       common_name: nir
       center_wavelength: 0.8328
       full_width_half_max: 0.106
+      openeo:gsd:
+        value:
+          - 10
+          - 10
+        unit: m
     - name: B8A
       common_name: nir08
       center_wavelength: 0.8647
       full_width_half_max: 0.021
+      openeo:gsd:
+        value:
+          - 20
+          - 20
+        unit: m
     - name: B09
       common_name: nir09
       center_wavelength: 0.9451
       full_width_half_max: 0.02
+      openeo:gsd:
+        value:
+          - 60
+          - 60
+        unit: m
     - name: B10
       common_name: cirrus
       center_wavelength: 1.3735
       full_width_half_max: 0.031
+      openeo:gsd:
+        value:
+          - 60
+          - 60
+        unit: m
     - name: B11
       common_name: swir16
       center_wavelength: 1.6137
       full_width_half_max: 0.091
+      openeo:gsd:
+        value:
+          - 20
+          - 20
+        unit: m
     - name: B12
       common_name: swir22
       center_wavelength: 2.2024
       full_width_half_max: 0.175
+      openeo:gsd:
+        value:
+          - 20
+          - 20
+        unit: m
   constellation:
     - sentinel-2
```

Comment:
`SCL` bands doesn't seem to exist in this collection (it's listed in datacube:bands).

Why are CLP, CLM etc. missing from `  eo:bands:`? (same for s2l2a)

Reply:
I originally didn't include them as I felt they are no eo:bands but more derived products. For the use case here they should be included but I forgot to add them here. I added them now.